### PR TITLE
fix issues about longformer package upgrades pytorch package

### PR DESCRIPTION
**Pull request by simpleoier**
Longformer depends on pytorch-lightning, which requires torch>=1.8.*. [link](https://github.com/PyTorchLightning/pytorch-lightning/blob/3e808b8a7002f13970c9d6f449fb2bb1042b9b98/requirements.txt#L4)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -91,6 +91,7 @@
         "black",
     ],
     "doc": [
+        "Jinja<=3.0.3",
```
Comment (sw005320) [inline setup.py:94]: We can leave the sphinx issue for now.
If the CI passes without the document build, we can merge this PR.

Comment (simpleoier) [reply setup.py]: OK. Then I think this PR is almost ready. A few minutes to finish the rest tests. However, the sphinx issue needs some more time to get fixed. Or we can just hope sphinx could fix [this issue](https://github.com/sphinx-doc/sphinx/issues/10291) soon.

Comment (sw005320) [reply setup.py]: I was expecting it would be fixed soon, but if it does not in the next week, I may ask you or @xinjli to do it.

**Comment by sw005320** [comment]
Thanks, @simpleoier, for taking care of it.
Also, can you ask some existing PRs to merge this change and rerun the CI check?
Rerunning the CI check is very important (that was my mistake...)

**Comment by simpleoier** [comment]
OK, I'll go through the existing PRs.

File: setup.py (hunk 1 above)
Comment (xinjli) [reply setup.py]: yeah, I can help with this if you want to fix it

**Pull request merged**